src/Localization.jsx

```diff
@@ -24,15 +25,9 @@ const Localization = React.createClass({
     };
   },
 
-  localize(key, ...values) {
-    const { messages } = this.props;
-    let string = get(messages, key, key);
-
-    if (values && string) {
-      string = util.format(string, ...values);
-    }
-
-    return string;
+  localize(key, values, defaultValue) {
+    const { messages, localize } = this.props;
+    return localize(messages, key, values, defaultValue);
```

Comment:
I feel like this should be `localize(messages, defaultValue, key, ...values)`, why not keep the spread at the end? Also, since we're changing this API we'll need to make sure `<Text />` is updated properly as well.